it/google-cloud-platform/build.gradle

```diff
@@ -72,13 +73,16 @@ dependencies {
 
     testImplementation project(path: ":sdks:java:testing:test-utils")
     testImplementation project(path: ":sdks:java:io:google-cloud-platform")
+    testImplementation project(path: ":sdks:java:extensions:protobuf", configuration: "testRuntimeMigration")
     testImplementation project(path: ":sdks:java:io:synthetic")
     testImplementation library.java.mockito_inline
     testImplementation project(path: ":sdks:java:extensions:google-cloud-platform-core", configuration: "testRuntimeMigration")
+    testImplementation project(path: ":runners:direct-java", configuration: "shadow")
```

Comment:
Why this is needed. There are "testRuntimeOnly" below.

If it is the case, then we can remove the line below.

Reply:
Removed the "testRuntimeOnly" line. We are using some classes from this dependency and IDE couldn't locate those without "testImplementation".